head_joint_pos_dict: 
  start: [0.0, 0.0]
  row_1: [0.0, 0.0]
  row_2: [0.0, 0.4]
  row_3: [0.0, 0.6]
  row_4: [0.0, 0.6]

